src/aggregator/integration/resend_stress_test.go

```diff
@@ -244,12 +241,6 @@ func TestResendAggregatedValueStress(t *testing.T) {
 	wg.Wait()
 	log.Sugar().Infof("done. Ran for %v\n", time.Since(start))
 
-	// Remove all the topic consumers before closing clients and servers. This allows to close the
-	// connections between servers while they still are running. Otherwise, during server shutdown,
-	// the yet-to-be-closed servers would repeatedly try to reconnect to recently closed ones, which
-	// results in longer shutdown times.
-	require.NoError(t, removeAllTopicConsumers(topicService, defaultTopicName))
```

Comment:
Why is this no longer needed? Wouldn't this make the shutdown slower?

Reply:
In this test, only a single server instance is launched, so shutdown doesn't cause problems (I guess this test was a copy of one of the multi-server tests).